app/views/users/_backnavbar.html.erb

```diff
@@ -0,0 +1,125 @@
+<div class="edit-navbar">
+  <div class="header mini-nav">
+    <nav class="navbar">
+      <div class="container-fluid">
+        <a class="navbar-brand" href="#"></a>
+        <ul class="navbar-nav float-end flex-row">
+          <% if user_signed_in? %>
+            <li class="nav-item dropdown" onmouseover="showDropdown()" onmouseout="hideDropdown()">
+              <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">| Welcome <%= current_user.full_name %>!</a>
+              <div class="dropdown-menu" aria-labelledby="navbarDropdown" id="myDropdown">
+                <a class="dropdown-item" href="/profile">Profile</a>
+                <a class="dropdown-item" href="/my-ads">My Ads</a>
```

Comment:
Seems like this whole div is being used again here, it has been used above in app/views/home/_navbar.html.erb, can we adjust it into a single partial?